src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -95,42 +102,61 @@ export class AccessibleBufferWidget extends DisposableStore {
 		this._xterm.raw.focus();
 	}
 
-	async show(): Promise<void> {
-		const commandDetection = this._capabilities.get(TerminalCapability.CommandDetection);
-		const fragment = !!commandDetection ? this._getShellIntegrationContent() : this._getAllContent();
-		const model = await this._getTextModel(URI.from({ scheme: AccessibleBufferConstants.Scheme, fragment }));
+	private async _updateContent(refresh?: boolean): Promise<ITextModel> {
+		const sb = new StringBuilder(10000);
+		if (refresh) {
+			sb.appendString(this._content);
+			sb.appendString(this._getContent(this._lastContentLength));
```

Comment:
lastContentLength being passed in a startLine is a little confusing

Reply:
yes wip